### DISCUSSION TITLE
Terminology: "Carbon emission" or "greenhouse gas (GHG) emission"?

### DISCUSSION THREAD

**Discussion by patricekrakow** [Green-Software-Foundation/sci#339] [SCI feedback]
Should you/we not replace the terms "carbon emission" by "greenhouse gas (GHG) emission"? For instance,

> This specification defines a methodology for calculating the rate of carbon emissions for a software system.

could/should be replaced by

> This specification defines a methodology for calculating the rate of greenhouse gas for a software system. 

It would be more consistent with the usage of "CO2 equivalent" in units of measure.

**Comment by atg-abhishek** [comment]
That's a great point and certainly worth discussing @patricekrakow! @seanmcilroy29 we can add to the next meeting agenda? 

In the meantime, thoughts on this @jawache @Henry-WattTime ?

**Comment by Henry-WattTime** [comment]
Asim: in training data we say that Carbon is CO2e. 
Options:
-Change it everywhere, might be more consistent
-Could also add a definition at the top that explains we mean CO2e when we  say Carbon 
In the title of the spec!
WG: to add a definition that clarifies what we mean when we say carbon. @Henry-WattTime to copy/adapt from https://learn.greensoftware.foundation/carbon-efficiency#carbon-and-co2eq

**Comment by Henry-WattTime** [comment]
Addressed in PR #346
